### PR TITLE
build: Add black

### PR DESCRIPTION
**Pull request by armrib**

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -208,37 +208,35 @@ def get_preamble() -> str:
...
-    'manual',
-)]
+latex_documents = [
+    (
+        master_doc,
+        f"{name}.tex",
+        f"{project} Documentation",
+        author,
+        "manual",
+    )
```
[the remark this answers is not in the document]
Comment (armrib) [reply docs/conf.py:231]: https://test-black.readthedocs.io/en/style-guide/style_guide/trailing_commas.html

**Review by louisrigot — APPROVED** [review]
Fine by me, but I'd wait for Wouter's approval to merge.

**Comment by louisrigot** [comment]
Oh, I didn't notice auto-merge was enabled 🤦

**Comment by armrib** [comment]
> Oh, I didn't notice auto-merge was enabled 🤦

It is fine @wouter-vdb did agree with it

**Pull request merged**